conf/application.conf

```diff
@@ -35,22 +35,19 @@ play.modules.enabled += "uk.gov.hmrc.play.bootstrap.AuditModule"
 # A metric filter must be provided
 play.modules.enabled += "uk.gov.hmrc.play.bootstrap.graphite.GraphiteMetricsModule"
 
-# Provides an implementation and configures all filters required by a Platform frontend microservice.
-play.modules.enabled += "uk.gov.hmrc.play.bootstrap.MicroserviceModule"
-play.http.filters = "uk.gov.hmrc.play.bootstrap.filters.MicroserviceFilters"
```

Comment:
This line is currently disabling all other filters by replacing the configuration value

The correct way to do this as of Play 2.6+ (I think) is to do `play.filters.enabled += WhateverFilter`, but the correct filters should already be enabled by the include of `backend.conf` at the top of this file.